test/ItemsRolesRegistryFacet/itemsRolesRegistryFacetTest.ts

```diff
@@ -233,7 +233,7 @@ describe("ItemsRolesRegistryFacet", async () => {
             roleAssignment.data,
           )
       })
-      it('should revert if grantor tries to update a grant with a nonce that its not theirs', async function () {
+      it.skip('should revert if grantor tries to update a grant with a nonce that its not theirs', async function () {
```

Comment:
```suggestion
      it('should revert if grantor tries to update a grant with a nonce that its not theirs', async function () {
```

Reply:
this test scenario will never happen anymore, because now the nonce is associated with grantorAddress, so it not allows cross-nonce test cases. If the nonce is equal different than a previous one, it will recognize it as a new deposit/grant